src/main/kotlin/study/racingCar/domain/Car.kt

```diff
@@ -0,0 +1,15 @@
+package study.racingCar.domain
+
+class Car(var movedSteps: Int = 0, var tryCounts: Int = 0) {
+
+    fun moveOneStep(trigger: Int = (0..9).random()): Car {
+        if (trigger >= MOVE_CONDITION) movedSteps++
+        tryCounts++
```

Comment:
중괄호를 써서 더 명확하게 하면 좋겠어요!

```suggestion
        if (trigger >= MOVE_CONDITION) {
            movedSteps++
        }
        tryCounts++
```